src/app/features/management/user-management-view/create-user-view/create-user-view.component.spec.ts

```diff
@@ -0,0 +1,272 @@
+      pass: pass,
+      isAdmin: isAdmin,
+    });
+    tick();
+    expect(component.form.valid).toBeFalse();
+  }));
+
+  it('should disable user creation without firstname', fakeAsync(() => {
+    component.form.setValue({
+      username: username,
+      firstName: '',
+      lastName: lastname,
+      pass: pass,
+      isAdmin: isAdmin,
+    });
+    tick();
+    expect(component.form.valid).toBeFalse();
+  }));
+
+  it('should disable user creation without lastname', fakeAsync(() => {
+    component.form.setValue({
+      username: username,
+      firstName: firstname,
+      lastName: '',
+      pass: pass,
+      isAdmin: isAdmin,
+    });
+    tick();
+    expect(component.form.valid).toBeFalse();
+  }));
+
+  it('should disable user creation without password', fakeAsync(() => {
+    component.form.setValue({
+      username: username,
+      firstName: firstname,
+      lastName: lastname,
+      pass: '',
+      isAdmin: isAdmin,
+    });
+    tick();
+    expect(component.form.valid).toBeFalse();
+  }));
+
+  it('should allow creation with all values set', fakeAsync(() => {
+    component.form.setValue({
+      username: username,
+      firstName: firstname,
+      lastName: lastname,
+      pass: pass,
+      isAdmin: isAdmin,
+    });
+    tick();
+    expect(component.form.valid).toBeTrue();
+  }));
+
+  describe('createUser', () => {
+    it('should fail without username', fakeAsync(() => {
+      component.form.setValue({
+        username: '',
+        firstName: firstname,
+        lastName: lastname,
+        pass: pass,
+        isAdmin: isAdmin,
+      });
+      tick();
+      expect(component.createUser).toThrowError();
+    }));
+
+    it('should fail without firstname', fakeAsync(() => {
+      component.form.setValue({
+        username: username,
+        firstName: '',
+        lastName: lastname,
+        pass: pass,
+        isAdmin: isAdmin,
+      });
+      tick();
+      expect(component.createUser).toThrowError();
+    }));
+
+    it('should fail without lastname', fakeAsync(() => {
+      component.form.setValue({
+        username: username,
+        firstName: firstname,
+        lastName: '',
+        pass: pass,
+        isAdmin: isAdmin,
+      });
+      tick();
+      expect(component.createUser).toThrowError();
+    }));
+
+    it('should fail without password', fakeAsync(() => {
+      component.form.setValue({
+        username: username,
+        firstName: firstname,
+        lastName: lastname,
+        pass: '',
+        isAdmin: isAdmin,
+      });
+      tick();
+      expect(component.createUser).toThrowError();
+    }));
+
+    it('should create user correctly', fakeAsync(() => {
+      component.form.setValue(({
+        username: username,
+        firstName: firstname,
+        lastName: lastname,
+        pass: pass,
+        isAdmin: isAdmin,
+      }));
+      tick();
+      spectator.inject(UserService).createUser.and.returnValue(of({
+        id:"Cb531BJu",
```

Comment:
```suggestion
        id: 'Cb531BJu',
```
better use random words btw for better readability